src/components/button/button.js

```diff
@@ -81,7 +81,7 @@ module.exports = class Button extends Control {
                 onMouseDown: this._onMouseDown.bind(this),
                 onMouseUp: this._onMouseUp.bind(this),
                 'aria-disabled': this.props.disabled
-            },
+            }, this.props.attrs),
```

Comment:
I think that it's better to pass handlers like simple props, not grouping them into attrs